corehq/apps/cloudcare/static/cloudcare/js/form_entry/fullform-ui.js

```diff
@@ -286,6 +286,17 @@ hqDefine("cloudcare/js/form_entry/fullform-ui", function () {
             return self.currentIndex() !== "0" && self.currentIndex() !== "-1" && !self.atFirstIndex();
         });
 
+        self.erroredLabels = ko.computed(function () {
+            var questions = getQuestions(self);
+            var erroredLabels = {};
+            for (var i = 0; i < questions.length; i++) {
+                if (questions[i].isLabel && !questions[i].isValid()) {
```

Comment:
I'd thought this would check `serverError` since that's what the code `webformsession` sets.